### PR TITLE
master_me: init at unstable-2022-09-28

### PR DESCRIPTION
**Pull request by magnetophon**
There are releases out, but they require
https://github.com/jpcima/faustpp, which is not in nixpkgs yet.
See: https://github.com/trummerschlunk/master_me/issues/91#issuecomment-1260727551

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/master_me/default.nix [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  buildInputs = [
+    libGL libX11 libXext libXrandr
+  ];
+
+  postPatch = ''
+    patchShebangs ./dpf/utils/
+    substituteInPlace ./dpf/utils/res2c.py \
+    --replace '/usr/bin/env python3' ${python3.interpreter}
+
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/master_me/default.nix:26]:
```suggestion
      --replace '/usr/bin/env python3' ${python3.interpreter}
```

**Comment by SuperSandro2000** [comment]
> There are releases out, but they require
> [jpcima/faustpp](https://github.com/jpcima/faustpp), which is not in nixpkgs yet.

Why not package it and use a stable release instead?

File: pkgs/applications/audio/master_me/default.nix [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{ lib, stdenv, fetchFromGitHub, libGL, libX11, libXext, libXrandr, libXinerama, libXcursor, freetype, alsa-lib, cmake, pkg-config, gcc-unwrapped, python3 }:
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/master_me/default.nix:1]:
```suggestion
{ lib, stdenv, fetchFromGitHub, libGL, libX11, libXext, libXrandr, libXinerama, libXcursor, freetype, alsa-lib, cmake, pkg-config, python3 }:
```
Please double check for other unused inputs

Comment (magnetophon) [reply pkgs/applications/audio/master_me/default.nix:1]: I'm working on it, but I ran a nix profile command, and now ``nix-env -f $NIXPKGS -iA master_me`` doesn't work anymore: ``error: profile '/nix/var/nix/profiles/per-user/bart/profile' is incompatible with 'nix-env'; please use 'nix profile' instead``.

Comment (magnetophon) [reply pkgs/applications/audio/master_me/default.nix]: Done.

**Comment by SuperSandro2000** [comment]
On the first look it seems like a standard python package and shouldn't be to hard. I would give it a try.

**Comment by magnetophon** [comment]
Superseded by https://github.com/NixOS/nixpkgs/pull/226341

**Pull request closed without merging**